zebra-network/src/protocol/external/message.rs

```diff
@@ -193,7 +193,12 @@ pub enum Message {
     /// content of a specific object, and is usually sent after
     /// receiving an `inv` packet, after filtering known elements.
     ///
+    /// `zcashd` sends a response containing all known items, in a contiguous
+    /// list. Missing blocks are silently skipped. Missing transaction hashes are
+    /// included in a single `NotFound` message following the transactions.
```

Comment:
```suggestion
```suggestion
    /// `zcashd` returns requested items in a single batch of messages.
    /// Missing blocks are silently skipped. Missing transaction hashes are
    /// included in a single `NotFound` message following the transactions.
    /// Other item or non-item messages can come before or after the batch.
```